---
layout: archive
title: "Publications"
permalink: /publications/
author_profile: true
---

{% include base_path %}

{% for post in site.publications reversed %}
  {% include archive-single.html %}
{% endfor %}

\*: Co-first, °: Co-corresponding

This page is no longer updated. You can find all my articles on <u><a href="https://scholar.google.de/citations?user=rafEIrMAAAAJ&hl=en">my Google Scholar profile</a>.</u>

2024 (4)
======
(47) T.M. Nordmann, H. Anderton, A. Hasegawa, L. Schweizer, P. Zhang, P. Stadler, A. Sinha, A. Metousis, F.A. Rosenberger, M. Zwiebel, T.K. Satoh, F. Anzengruber, **M.T. Strauss**, M.C. Tanzer, Y. Saito, T. Gong, M. Thielert, H. Kimura, N. Silke, E.H. Rodriguez, G. Restivo, H.H. Nguyen, A. Gross, L. Feldmeyer, L. Joerg, M.P. Levesque, P.J. Murray, S. Ingen-Housz-Oro, A. Mund, R. Abe, J. Silke, C. Ji, L.E. French, M. Mann°  
*Spatial proteomics identifies JAKi as treatment for a lethal skin disease*  
[**Nature**](https://doi.org/10.1038/s41586-024-08061-0) (2024).

---
(46) S. Kabatnik, F. Post, L. Drici, A.S. Bartels, **M.T. Strauss**, X. Zheng, G.I. Madsen, A. Mund, F.A. Rosenberger, J. Moreira, M. Mann°  
*Spatial characterization and stratification of colorectal adenomas by deep visual proteomics*  
[**iScience**](https://doi.org/10.1016/j.isci.2024.110620) (2024).

---
(45) P. Beltrao, T. Van Den Bossche, R. Gabriels, T. Holstein, T. Kockmann, A. Nameni, C. Panse, R. Schlapbach, L. Lautenbacher, M. Mattanovich, A. Nesvizhskii, B. Van Puyvelde, J. Scheid, V. Schwämmle, **M.T. Strauss**, A. Klimovskaia Susmelj, M. The, H. Webel, M. Wilhelm, D. Winkelhardt, W.E. Wolski, M. Xi  
*Proceedings of the EuBIC-MS developers meeting 2023*  
[**Journal of Proteomics**](https://doi.org/10.1016/j.jprot.2024.105246) (2024).

---
(44) **M.T. Strauss°**, I. Bludau, W. Zeng, E. Voytik, C. Ammar, J.P. Schessner, I. Rajesh, M. Gill, F. Meier, S. Willems, M. Mann° 
*AlphaPept: a modern and open framework for MS-based proteomics*  
[**Nature Communications**](https://doi.org/10.1038/s41467-024-46485-4) (2024) 

2023 (5)
======
(43) A.C. Michaelis, A-D. Brunner, M. Zwiebel, F. Meier, **M.T. Strauss**, I. Bludau, M. Mann° 
*The social and structural architecture of the yeast protein interactome*  
[**Nature**](https://doi.org/10.1038/s41586-023-06739-5) (2023). 

---
(42) F.A. Rosenberger, M. Thielert, **M.T. Strauss**,  L. Schweizer, C. Ammar, S.C. Mädler, A. Metousis, P. Skowronek, M. Wahle, K. Madden, J. Gote-Schniering, A. Semenova, H.B. Schiller, E. Rodriguez, T.M. Nordmann, A. Mund, M. Mann° 
*Spatial single-cell mass spectrometry defines zonation of the hepatocyte proteome*  
[**Nature Methods**](https://doi.org/10.1038/s41592-023-02007-6) (2023). 

---
(41) E. Krismer, I. Bludau, **M.T. Strauss°**, M. Mann° 
*AlphaPeptStats: an open-source Python package for automated and scalable statistical analysis of mass spectrometry-based proteomics*  
[**Bioinformatics**](https://doi.org/10.1093/bioinformatics/btad461) (2023). 

---
(40) S. Lobentanzer°, P. Aloy, J. Baumbach, B. Bohar, V.J. Carey, P. Charoentong, K. Danhauser, T. Doğan, J. Dreo, I. Dunham, E. Farr, A. Fernandez-Torras, B.M. Gyori, M. Hartung, C.T. Hoyt, C. Klein, T. Korcsmaros, A. Maier, M. Mann, D. Ochoa, E. Pareja-Lorente, F. Popp, M. Preusse, N. Probul, B. Schwikowski, B. Sen, **M.T. Strauss**, D. Turei, E. Ulusoy, D. Waltemath, J.A.H. Wodke, J. Saez-Rodriguez° 
*Democratizing knowledge representation with BioCypher*  
[**Nature Biotechnology**](https://doi.org/10.1038/s41587-023-01848-y) (2023). 

---
(39) S.C.M. Reinhardt\*, L. Masullo\*, I. Baudrexel\*, P.R. Steen\*, R. Kowaleswki, A.E. Eklund, S. Strauss, E.M. Unterauer, T. Schlichthaerle,  **M.T. Strauss**,  C. Klein, R. Jungmann°  
*Ångström-resolution fluorescence microscopy*  
[**Nature**](https://doi.org/10.1038/s41586-023-05925-9) (2023).  

2022 (6)
======
(38) F. Torun, S. Virreira Winter, S. Doll, F.M. Riese, A. Vorobyev, J.B. Mueller-Reif, **M.T. Strauss°**, 
*Transparent Exploration of Machine Learning for Biomarker Discovery from Proteomics and Omics Data.*
[**Journal of Proteome Research**](https://doi.org/10.1021/acs.jproteome.2c00473) (2022).

---
(37) W. Zeng, X. Zhou, S. Willems, C. Ammar, M. Wahle, I. Bludau, E. Voytik, **M.T. Strauss**, M. Mann°, 
*AlphaPeptDeep: a modular deep learning framework to predict peptide properties for proteomics.*
[**Nature Communications**](https://doi.org/10.1038/s41467-022-34904-3) (2022).

---
(36) **M.T. Strauss°**, 
*Picasso-server: a community-based, open-source processing framework for super-resolution data.*
[**Communications Biology**](https://doi.org/10.1038/s42003-022-03909-5) (2022).

---
(35) L. Niu, M. Thiele, P.E. Geyer, D.N. Rasmussen, H.E. Webel, A. Santos, R. Gupta, F. Meier, **M.T. Strauss**, M. Kjaergaard, K. Lindvig, S. Jacobsen, S. Rasmussen, T. Hansen, A. Krag, M. Mann°, 
*Noninvasive proteomic biomarkers for alcohol-related liver disease.*
[**Nature Medicine**](https://doi.org/10.1038/s41591-022-01850-y) (2022).

---
(34) I. Bludau, S. Willems, W. Zeng, **M.T. Strauss**, F.M. Hansen, M.C. Tanzer, O. Karayel, B.A. Schulman, M. Mann°,  
*The structural context of posttranslational modifications at a proteome-wide scale*  
[**PLoS Biol**](https://doi.org/10.1371/journal.pbio.3001636) (2022). 20(5): e3001636.

---
(33) A. Santos°, A.R. Colaço, A.B. Nielsen, L. Niu, **M.T. Strauss**, P.E. Geyer, N.J. Wewer Albrechtsen, F. Mundt, L.J. Jensen, M. Mann°,  
*A knowledge graph to interpret clinical proteomics data*  
[**Nature Biotechnology**](https://doi.org/10.1038/s41587-021-01145-6) (2022).

2021 (7)
======
(32) E. Voytik, I. Bludau, S. Willems, F.M. Hansen, A. Brunner, **M.T. Strauss**, M. Mann°,  
*AlphaMap: an open-source python package for the visual annotation of proteomics data with sequence specific knowledge*  
[**Bioinformatics**](https://doi.org/10.1093/bioinformatics/btab674) (2021).

---
(31) S. Willems, E. Voytik, P. Skowronek, **M.T. Strauss**, M. Mann°,  
*AlphaTims: Indexing Trapped Ion Mobility Spectrometry–TOF Data for Fast and Easy Accession and Visualization*  
[**MCP**](https://doi.org/10.1016/j.mcpro.2021.100149) (2021).

---
(30) M. Mann°, C. Kumar°, W. Zeng°, **M.T. Strauss°**,  
*Artificial intelligence for proteomics and biomarker discovery*  
[**Cell Systems**](https://doi.org/10.1016/j.cels.2021.06.006) (2021). 12(8): 759-770

---
(29) P.E. Geyer\*°, F.M. Arend, S. Doll, M.L. Louiset, S. Virreira Winter, J.B. Müller-Reif, F.M. Torun, M. Weigand, P. Eichhorn, M. Bruegel, **M.T. Strauss**, L.M. Holdt, M. Mann, D. Teupser°   
*High-resolution serum proteome trajectories in COVID-19 reveal patient-specific seroconversion*  
[**EMBO Mol Med**](https://doi.org/10.15252/emmm.202114167) (2021). e14167  

---
(28) F. Meier\*, N.D. Köhler\*, A. Brunner\*, J.H. Wanka, E. Voytik, **M.T. Strauss**, F.J. Theis°, M. Mann°   
*Deep learning the collisional cross sections of the peptide universe from a million experimental values*  
[**Nature Communications**](https://www.nature.com/articles/s41467-021-21352-8) (2021). 12: 1185  

---
(27) L.S. Fischer\*, C. Klingner\*, T. Schlichthaerle\*, **M.T. Strauss**, R. Böttcher, R. Fässler°, R. Jungmann°, C. Grashoff°  
*Quantitative single-protein imaging reveals molecular complex formation of integrin, talin, and kindlin during cell adhesion*  
[**Nature Communications**](https://www.nature.com/articles/s41467-021-21142-2) (2021). 12: 919  

---
(26) S.V. Winter\*, O. Karayel\*,  **M.T. Strauss**,  S. Padmanabhan, M Surface, K. Merchant, R.N. Alcalay, M. Mann°  
*Urinary proteome profiling for stratifying patients with familial Parkinson's disease*  
[**EMBO Mol Med**](https://doi.org/10.15252/emmm.202013257) (2021). e13257  

2020 (4)
======
(25) S.F.J. Wickham, A. Auer, J. Min, N. Ponnuswamy, J.B. Woehrstein, F. Schueder, **M.T. Strauss**, J. Schnitzbauer, B. Nathwani, Z. Zhao, S.D. Perrault, J. Hahn, S. Lee, M.M. Bastings, S.W. Helmig, A.L Kodal, P. Yin, R. Jungmann°, W.M. Shih°   
*Complex multicomponent patterns rendered on a 3D DNA-barrel pegboard*  
[**Nature Communications**](https://dx.doi.org/10.1038/s41467-020-18910-x) (2020). 11: 5768

---
(24) J.B. Mueller \*, P.E. Geyer\*, A.R. Colaço, P.V. Treit, **M.T. Strauss**, M. Oroshi, S. Doll, S.V. Winter, J.B. Bader, N. Koehler, F. Theis, A. Santos, M. Mann  
*The proteome landscape of the kingdoms of life*  
[**Nature**](https://doi.org/10.1038/s41586-020-2402-x) (2020). 582, 592–596

---
(23) J.M. Bader\*, P.E. Geyer\*, J.B. Muller, **M.T. Strauss**, M. Koch, F. Leypoldt, P. Koertvelyessy, D. Bittner, C.G. Schipke, E.I. Incesoy, O. Peters, N. Deigendesch, M. Simons, M.K. Jensen, H. Zetterberg, M. Mann  
*Proteome profiling in cerebrospinal fluid reveals novel biomarkers of Alzheimer's disease*  
[**Molecular Systems Biology**](https://doi.org/10.15252/msb.20199356) (2020). 16: e9356

---
(22) A. Auer, **M.T. Strauss**, S. Strauss, R. Jungmann  
*nanoTRON: a Picasso module for MLP-based classification of super-resolution data*  
[**Bioinformatics**](https://dx.doi.org/10.1093/bioinformatics/btaa154) (2020). 20(09): 6732-6737

2019 (5)
======
(21) F. Schueder, J. Stein, F. Stehr, A. Auer, B. Sperl, **M.T. Strauss**, P. Schwille, R. Jungmann  
*An order of magnitude faster DNA-PAINT imaging by optimized sequence design and buffer conditions*  
[**Nature Methods**](https://dx.doi.org/10.1038/s41592-019-0584-7) (2019). 16: 1101–1104 

---
(20) C. Böger, A. Hafner, T. Schlichthaerle, **M.T. Strauss**, S. Malkusch, U. Endesfelder, R. Jungmann, E.M. Schuman°, M. Heilemann°  
*Super-resolution imaging and estimation of protein copy numbers at single synapses with DNA-point accumulation for imaging in nanoscale topography*  
[**Neurophotonics**](https://dx.doi.org/10.1117/1.NPh.6.3.035008) (2019). 6(3): 035008

---
(19) T. Schlichthaerle\*, **M.T. Strauss\***, F. Schueder\*, A. Auer, B. Nijmeijer, M. Kueblbeck, V.J. Sabinina, J.V. Thevathasan, J. Ries, J. Ellenberg, R. Jungmann  
*Direct visualization of single nuclear pore complex proteins using genetically-encoded probes for DNA-PAINT*  
[**Angewandte Chemie Int. Ed.**](https://dx.doi.org/10.1002/anie.201905685) (2019). 58: 13004-13008

---
(18) O.K. Wade\*, J.B. Woehrstein\*, P.C. Nickels\*, S. Strauss\*, F. Stehr, J. Stein, F. Schueder, **M.T. Strauss**, M. Ganji, J. Schnitzbauer, H. Grabmayr, P. Yin, P. Schwille, R. Jungmann  
*124-color super-resolution imaging by engineering DNA-PAINT blinking kinetics*  
[**Nano Letters**](https://dx.doi.org/10.1021/acs.nanolett.9b00508) (2019). 19(4): 2641-2646

---
(17) S. Kempter, A. Khmelinskaia, **M.T. Strauss**, P. Schwille, R. Jungmann, T. Liedl, W. Bae  
*Single Particle Tracking and Super-Resolution Imaging of Membrane-Assisted Stop-and-Go Diffusion and Lattice Assembly of DNA Origami*  
[**ACS Nano**](https://dx.doi.org/10.1021/acsnano.8b04631) (2019). 13(2): 996–1002

2018 (8)
======
---
(16) H. Heydarian, F. Schueder, **M.T. Strauss**, B. van Werkhoven, M. Fazel, K.A. Lidke, R. Jungmann, S. Stallinga, B. Rieger  
*Template-free 2D particle fusion in localization microscopy*  
[**Nature Methods**](https://dx.doi.org/10.1038/s41592-018-0136-6) (2018). 15: 781–784

---
(15) A. Auer, T. Schlichthaerle, J.B. Woehrstein, F. Schueder, **M.T. Strauss**, H. Grabmayr, R. Jungmann  
*Nanometer-scale multiplexed super-resolution imaging with an economic 3D-DNA-PAINT microscope*  
[**ChemPhysChem**](https://dx.doi.org/10.1002/cphc.201800630) (2018). 19(22): 3024–3034 

---
(14) S. Strauss, P.C. Nickels, **M.T. Strauss**, V.J. Sabinina, J. Ellenberg, J.D. Carter, S. Gupta, N. Janjic, R. Jungmann  
*Modified aptamers enable quantitative sub-10-nm cellular DNA-PAINT imaging*  
[**Nature Methods**](https://dx.doi.org/10.1038/s41592-018-0105-0) (2018). 15: 685–688

---
(13) T. Schlichthaerle, A.S. Eklund, F. Schueder, **M.T. Strauss**, C. Tiede, A. Curd, J. Ries, M. Peckham, D.C. Tomlinson, R. Jungmann  
*Site-specific labeling of Affimers for DNA-PAINT microscopy*  
[**Angewandte Chemie Int. Ed.**](https://doi.org/10.1002/anie.201804020) (2018). 57: 11060-11063

---
(12) N.S. Deußner-Helfmann, A. Auer, **M.T. Strauss**, S. Malkusch, M.S. Dietz, H. Barth, R. Jungmann°, M. Heilemann°  
*Correlative Single-Molecule FRET and DNA-PAINT Imaging*  
[**Nano Letters**](https://dx.doi.org/10.1021/acs.nanolett.8b02185) (2018). 18(7): 4626-4630

---
(11) **M.T. Strauss\***, F. Schueder\*, D. Haas\*, P.C. Nickels, R. Jungmann    
*Quantifying absolute addressability in DNA origami with molecular resolution*  
[**Nature Communications**](http://dx.doi.org/10.1038/s41467-018-04031-z) (2018). 9: 1600

---
(10) J. Mücksch\*, P. Blumhardt\*, **M.T. Strauss**, E.P. Petrov, R. Jungmann, P. Schwille  
*Quantifying reversible surface binding via surface-integrated FCS*  
[**Nano Letters**](http://dx.doi.org/10.1021/acs.nanolett.8b00875) (2018). 18(5): 3185–3192

---
(9) S. Baenfer, D. Schneider, J. Dewes, **M.T. Strauss**, S. Freibert, T. Heimerl, U.G. Maier, H. Elsaesser, R. Jungmann, R. Jacob  
*A molecular mechanism to recruit galectin-3 into multivesicular bodies for polarized exosomal secretion*  
[**PNAS**](https://doi.org/10.1073/pnas.1718921115) (2018). 115(19): E4396-E4405

2017 (6)
======
---
(8) F. Schueder, J. Lara-Guitérrez, B.J. Beliveau, S.K. Saka, H.M. Sasaki, J.B. Woehrstein, **M.T. Strauss**, H. Grabmayr, P. Yin°, R. Jungmann°  
*Multiplexed 3D super-resolution imaging of whole cells using Spinning Disk Confocal Microscopy and DNA-PAINT*  
[**Nature Communications**](http://dx.doi.org/10.1038/s41467-017-02028-8) (2017). 8: 2090

---
(7) A. Auer, **M.T. Strauss**, T. Schlichthaerle, R. Jungmann  
*Fast, background-free DNA-PAINT imaging using FRET-based probes*  
[**Nano Letters**](http://dx.doi.org/10.1021/acs.nanolett.7b03425) (2017). 17(10): 6428-6434  

---
(6) L.L. Ong, N. Hanikel, O.K. Yaghi, C. Grun, **M.T. Strauss**, P. Bron, J. Lai-Kee-Him, F. Schueder, B. Wang, P. Wang, J.Y. Kishi, C.A. Myhrvold, A. Zhu, R. Jungmann, G. Bellot°, Y. Ke°, P. Yin°  
*Programmable self-assembly of three-dimensional nanostructures from 10,000 unique components*  
[**Nature**](http://dx.doi.org/10.1038/nature24648) (2017). 552: 72-77  

---
(5) J.B. Woehrstein\*, **M.T. Strauss\***, L.L. Ong, B. Wei, D.Y. Zhang, R. Jungmann°, P. Yin°  
*Sub-100-nm metafluorophores with digitally tunable optical properties self-assembled from DNA*  
[**Science Advances**](http://dx.doi.org/10.1126/sciadv.1602128) (2017). 3: e1602128  

---
(4) J. Schnitzbauer\*, **M.T. Strauss\***, T. Schlichthaerle, F. Schueder, R. Jungmann  
*Super-Resolution Microscopy with DNA-PAINT*  
[**Nature Protocols**](http://dx.doi.org/10.1038/nprot.2017.024) (2017). 12: 1198-1228  

---
(3) F. Schueder, **M.T. Strauss**, D. Hoerl, J. Schnitzbauer, T. Schlichthaerle, S. Strauss, P. Yin, H. Harz, H. Leonhardt, R. Jungmann  
*Universal Super-Resolution Multiplexing by DNA Exchange*  
[**Angewandte Chemie Int. Ed.**](http://doi.org/10.1002/anie.201611729) (2017). 56: 4052-4055  

2016 (1)
======
---
(2) T. Schlichthaerle, **M.T. Strauss**, F. Schueder, J.B. Woehrstein, R. Jungmann  
*DNA nanotechnology and fluorescence applications*  
[**Current Opinion in Biotechnology**](http://dx.doi.org/10.1016/j.copbio.2015.12.014) (2016). 39: 41-47

2015 (1)
======
---
(1) J.B. Knudsen. L. Liu, A.L.B. Kodal, M. Madsen, Q. Li, J. Song, J.B. Woehrstein, S.F.J. Wickham, **M.T. Strauss**, F. Schueder, J. Vinther, A. Krissanaprasit, D. Gudnason, A.A.A. Smith, R. Ogaki, A.N. Zelikin, F. Besenbacher, V. Birkedal, P. Yin, W.M. Shih. R. Jungmann, M. Dong°, K.V. Gothelf°  
*Routing of individual polymers in designed patterns*  
[**Nature Nanotechnology**](http://dx.doi.org/10.1038/nnano.2015.190>) (2015). 11: 892-898